core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -17,6 +17,7 @@
 package io.grpc;
 
 import com.google.common.base.Preconditions;
+import io.grpc.internal.GrpcUtil;
```

Comment:
The (Bazel) CI is failing because of this. We have a rule not to have io.grpc depend on io.grpc.internal (long-term we have hopes of splitting out a grpc-api which just contains io.grpc).

Reply:
Fixed. (Since you can't trigger the CI, I figured it would just be easiest if I fixed this up.)